src/bindings/python/src/openvino/runtime/convert_impl.py

```diff
@@ -0,0 +1,821 @@
+        if argv.saved_model_tags is not None:
+            if ' ' in argv.saved_model_tags:
+                raise Error('Incorrect saved model tag was provided. Specify "saved_model_tags" with no spaces in it')
+            argv.saved_model_tags = argv.saved_model_tags.split(',')
+
+    python_api_params_parsing(argv)
+    argv.output = argv.output.split(',') if argv.output else None
+    argv.layout_values = get_layout_values(argv.layout, argv.source_layout, argv.target_layout)
+    mean_values = parse_tuple_pairs(argv.mean_values)
+    scale_values = parse_tuple_pairs(argv.scale_values)
+    mean_scale = get_mean_scale_dictionary(mean_values, scale_values, argv.input)
+    argv.mean_scale_values = mean_scale
+
+    log.debug("Placeholder shapes : {}".format(argv.placeholder_shapes))
+
+    return argv
+
+
+def check_fallback(argv: argparse.Namespace):
+    fallback_reasons = {}
+
+    # Some frontend such as PDPD does not have legacy path so it has no reasons to fallback
+    if not any(deduce_legacy_frontend_by_namespace(argv)):
+        return fallback_reasons
+
+    if argv.use_new_frontend:
+        return fallback_reasons
+
+    fallback_reasons['extensions'] = legacy_extensions_used
+    fallback_reasons['transformations_config'] = legacy_transformations_config_used
+    fallback_reasons['tensorflow_custom_operations_config_update'] = tensorflow_custom_operations_config_update_used
+
+    reasons = [reason for reason, is_applicable in fallback_reasons.items() if is_applicable(argv)]
+    return reasons
+
+
+def update_fallback_with_conversion_error(use_new_frontend: bool, is_tf: bool, ex_msg: str, fallback_reasons: list):
+    import re
+    if not is_tf:
+        # this sort of fallback is only used by TensorFlow Frontend
+        return False
+
+    if use_new_frontend:
+        # this option forces to use new TensorFlow Frontend
+        # so it is not possible for the fallback
+        return False
+
+    # for TensorFlow FE we have a set of operations that should lead to the fallback to the legacy
+    conversion_error_re = r"^(\[TensorFlow\ Frontend\]\ Internal\ error\,\ no\ translator\ found\ for\ operation\(s\)\:\ )((\w+)(\,\ \w+)*)$"
+    conversion_error_match = re.findall(conversion_error_re, ex_msg, re.MULTILINE)
+    all_fallback_operations = [
+        # corresponds to TF1 While operation
+        "TensorArrayScatterV3", "TensorArrayV3", "TensorArraySizeV3", "TensorArrayGatherV3",
+        "LoopCond", "Enter", "NextIteration", "Exit",
+        # corresponds to TF1 If and TF1 While operations
+        "Switch", "Merge",
+        # corresponds to operations with complex tensors
+        "FFT", "FFT2D", "FFT3D", "IFFT", "IFFT2D", "IFFT3D",
+        "RFFT", "RFFT2D", "RFFT3D", "IRFFT", "IRFFT2D", "IRFFT3D",
+        "Complex", "ComplexAbs", "Real", "Imag",
+    ]
+    if len(conversion_error_match) < 1 or len(conversion_error_match[0]) != 4:
+        # no match for the fallback by unsupported operation
+        return False
```

Comment:
I think there should not be fallback to legacy TF FE

Reply:
Yes, if fallback is needed we get the following error:
"The provided model cannot be converted with new frontend, as fallback to legacy is needed. Please try to install openvino-dev and use convert_model() from openvino.tools.mo."